src/test/lib/hyrise_test.cpp

```diff
@@ -0,0 +1,48 @@
+#include "base_test.hpp"
+#include "gtest/gtest.h"
+
+#include "./utils/plugin_test_utils.hpp"
+#include "concurrency/transaction_manager.hpp"
+#include "hyrise.hpp"
+#include "storage/storage_manager.hpp"
+#include "storage/table.hpp"
+#include "types.hpp"
+#include "utils/plugin_manager.hpp"
+
+namespace opossum {
+
+class HyriseTest : public BaseTest {
+ protected:
+  void SetUp() override { Hyrise::reset(); }
+
+  std::unordered_map<PluginName, PluginHandleWrapper>& get_plugins() {
```

Comment:
You could add a comment, why we need this function. Private member access, I guess?